README.md

```diff
@@ -73,6 +73,29 @@ void main() {
 }
 ```
 
+## Actions
+
+Dashbook also provides a way for easily calling methods from its UI, these callbacks can be used for more complex examples which demands user interaction.
+
+For example, a `Dialog` is something that isn't direclty rendered on a page, but rather show upon an action, an example for a `CustomDialog` widget could be achieved on Dashbook by using the following code:
```

Comment:
Maybe split up this line so that it isn't so long?

Reply:
Directly* shows*

Reply:
I need to split this whole file :P

Fixed the typos, thanks!